docs/feature_rgb_matrix.md

```diff
@@ -742,6 +742,27 @@ void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
 }
 ```
 
+Layer indicator with only configured keys:
+```c
+void rgb_matrix_indicators_user(void) {
+    if (get_highest_layer(layer_state) > 0) {
+        for (uint8_t row = 0; row < MATRIX_ROWS; ++row) {
+            for (uint8_t col = 0; col < MATRIX_COLS; ++col) {
+
+                uint8_t index = g_led_config.matrix_co[row][col];
+                uint8_t layer = get_highest_layer(layer_state);
+
+                if (index >= led_min && index <= led_max && index != NO_LED &&
+                keymap_key_to_keycode(layer, (keypos_t){col,row}) > KC_TRNS) {
+                    rgb_matrix_set_color(index, RGB_LAYER);
```

Comment:
```suggestion
                    rgb_matrix_set_color(index, 255, 0, 0);
```
RGB_LAYER isn't declared, maybe it's just best to set them to red for the example?